src/Data/Data.php

```diff
@@ -114,14 +95,12 @@ public static function read(string $file, string $type = null): array
 	 * Writes data to a file;
 	 * the data handler is automatically chosen by
 	 * the extension if not specified
-	 *
-	 * @param string $file
-	 * @param mixed $data
-	 * @param string $type
-	 * @return bool
 	 */
-	public static function write(string $file = null, $data = [], string $type = null): bool
-	{
+	public static function write(
+		string|null $file = null,
```

Comment:
I think this is worth a breaking change. `fileinfo()` (which is called by `F::extension()`) throws a deprecation warning if it gets passed `null`. Also I really cannot see the use case to write to a file `null`.

```suggestion
		string $file,
```